### PR TITLE
SQL - Quiz Questions

### PR DESCRIPTION
**Pull request by sebaraba**
# What is done in this PR:
- [x] Quiz Question SELECT;
- [x] Quiz Question WHERE;
- [x] Quiz Question WHERE2;
- [x] Quiz Question WHERE3;
- [x] Quiz Question ORDER BY;
- [x] Quiz Question INNER JOIN;
- [x] Quiz Question FULL JOIN;
- [x] Quiz Question LEFT AND RIGHT JOIN;
- [x] Quiz Question multiple JOINS;
- [x] Union
- [x] Quiz Question CREATE INDEX;
- [x] Quiz Question When to add indexes;
# To Do:

- [x] Review on UNION;
- [x] Review on CREATE INDEX;
- [x] Review on when to add indexes;

### CODE REVIEW

**Comment by enki-hq** [comment]
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/sql/dql/basic-queries/order-by-clause.md?ref=6399dc000a5c790dc1780cc0bff7ba0dec58a10e
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/sql/dql/basic-queries/select.md?ref=6399dc000a5c790dc1780cc0bff7ba0dec58a10e
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/sql/dql/basic-queries/where-2.md?ref=6399dc000a5c790dc1780cc0bff7ba0dec58a10e
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/sql/dql/basic-queries/where-3.md?ref=6399dc000a5c790dc1780cc0bff7ba0dec58a10e
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/sql/dql/basic-queries/where.md?ref=6399dc000a5c790dc1780cc0bff7ba0dec58a10e

File: sql/dql/basic-queries/order-by-clause.md [hunk 1]
```diff
@@ -132,3 +132,28 @@ count | region_id
 * region_id
 * ORDER BY
 * SUM
+
+---
+## Quiz
+
+headline: Can you order it?
```
Comment (lizTheDeveloper) [inline sql/dql/basic-queries/order-by-clause.md]: Can you order a table with SQL?

File: sql/dql/basic-queries/where-2.md [hunk 2]
```diff
@@ -102,3 +102,35 @@ Select all the columns from the `pokemon` table, keeping only the rows where the
 * AND
 * 'ivysaur'
 * 'bulbasaur'
+
+---
+## Quiz
+
+headline: Get records that are IN BETWEEN.
```
Comment (lizTheDeveloper) [inline sql/dql/basic-queries/where-2.md]: Can you select rows between two values in SQL?

File: sql/dql/basic-queries/where-3.md [hunk 3]
```diff
@@ -91,3 +91,27 @@ Select all the columns from the `pokemon` table, keeping only the rows where the
 * AND
 * FIRST
 * 'g'
+
+---
+## Quiz
+
+headline: Do you know WHERE to put `%`?
```
Comment (lizTheDeveloper) [inline sql/dql/basic-queries/where-3.md]: Do you know how to match a pattern in SQL?

File: sql/dql/basic-queries/where.md [hunk 4]
```diff
@@ -73,3 +73,33 @@ Select all the rows from the `pokemon` table where the name is exactly equal to
 * pokemon
 * IS
 * 'braixen'
+
+---
+## Quiz
+
+headline: WHERE to find it?
```
Comment (lizTheDeveloper) [inline sql/dql/basic-queries/where.md]: Can you find a particular entry in a table with SQL?

File: sql/dql/joins/full-join.md [hunk 5]
```diff
@@ -83,3 +83,57 @@ pokemon_type.type_id = type.id
 * `ON`
 * `LEFT AND RIGHT JOIN`
 * `OUTER JOIN`
+
+---
+## Quiz
+
+headline: Take it all!
```
Comment (lizTheDeveloper) [inline sql/dql/joins/full-join.md]: Can you select data from two related tables in SQL?

File: sql/dql/joins/inner-join.md [hunk 6]
```diff
@@ -129,3 +129,31 @@ What's the sets theory equivalent of `INNER JOIN`?
...
+
+---
+## Quiz
+
+headline: Can you see the match?
+
+question: |
+
+  Consider `pokemon` and `pokemon_species` tables. The first one has two columns `height` and `weight`. The latter has a boolean valued column `is_baby`. Select the query that will get the name, height and width of
+  all pokemons that are babys:
```
Comment (lizTheDeveloper) [inline sql/dql/joins/inner-join.md]: babies*

File: sql/dql/joins/inner-join.md [hunk 7]
```diff
@@ -129,3 +129,31 @@ What's the sets theory equivalent of `INNER JOIN`?
 * union
 * difference
 * cartesian product
+
+---
+## Quiz
+
+headline: Can you see the match?
```
Comment (lizTheDeveloper) [inline sql/dql/joins/inner-join.md]: Can you filter one table based on data in another table in SQL?

**Comment by enki-hq** [comment]
Invalid section title on line 111:      1 |     1 |      7 |     69 |              64 |                  1 | bulbasaur

**Pull request merged**